data/ecosystems/s/starknet.toml

```diff
@@ -7359,6 +7354,9 @@ url = "https://github.com/yasmeen/starknet-app"
 [[repo]]
 url = "https://github.com/yekssin/starknet"
 
+[[repo]]
+url = "https://github.com/yetanotherco/yet-another-bridge"
```

Comment:
no need to add if you are creating a new ecosystem that includes this URL and adding such eco as a sub-eco of this eco